### DISCUSSION TITLE
How do I show the UI when it doesn’t pop up

### DISCUSSION THREAD

**Discussion by Beastgamer1M** [dnschneid/crouton#5118] [Q&A]
I've installed and uninstalled crouton dozens of times, the issue is that I can't see the UI upon powering the machine, I'm on an Acer Spin C720 Chromebook and this hasn't happened to me the first few experiences on Crouton, please help.

**Comment by 0nesnow** [comment]
Yeah so I think the problem is that you are installing xenial for some reason. I am having the same problem
You just have to install another version of Ubuntu try this commond in chronos@localhost

> sudo crouton -r focal -t xfce

Wait for it to install after that follow the normal stuff you do username and password

Then

> Sudo startxfce4


It should work

**Reply by Beastgamer1M** [reply]
> Yeah so I think the problem is that you are installing xenial for some reason. I am having the same problem
> 
> You just have to install another version of Ubuntu try this commond in chronos@localhost
> 
> 
> 
> > sudo crouton -r focal -t xfce
> 
> 
> 
> Wait for it to install after that follow the normal stuff you do username and password
> 
> 
> 
> Then
> 
> 
> 
> > Sudo startxfce4
> 
> 
> 
> 
> 
> It should work

I hope it does, I'm using unity so I'm doing sudo startunity
